tools/READMEs/make_readme.py

```diff
@@ -0,0 +1,40 @@
+from jinja2 import Environment, FileSystemLoader
+import json
+import os
+import sys
+
+if len(sys.argv) <= 1:
+    raise Exception("You should provide the path to the app as first arg")
+
+app = sys.argv[1]
+
+if not os.path.exists(app):
+    raise Exception("App path provided doesn't exists ?!")
+
+env = Environment(loader=FileSystemLoader('./templates'))
+
+for lang, lang_suffix in [("en", ""), ("fr", "_fr")]:
```

Comment:
Move it to a function and make it possible to import this file

Reply:
Done in https://github.com/YunoHost/apps/pull/1136/commits/231890575060a9b4d6a5c3a56978ad0f9c075617